### PR TITLE
Enhance datavalidator for template fields

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: docs/datavalidator/check_json.py [hunk 1]
```diff
@@ -125,9 +220,25 @@ def check_collections(self) -> None:
             raise CheckException(err)
 
     def check_model(self, collection: str, model: Dict[str, Any]) -> None:
-        model_fields = set(x for x in model.keys() if "$" not in x)
+        errors = self.check_normal_fields(model, collection)
+
+        if not errors:
```
Comment (jsangmeister) [inline docs/datavalidator/check_json.py:225]: Isn't it better to check everything, regardless if there were errors?

**Review by jsangmeister — APPROVED** [review]
Looks good on first sight, did not test it further. Will https://github.com/OpenSlides/OpenSlides/pull/6062 be also fixed with this?

**Comment by jsangmeister** [comment]
Also https://github.com/OpenSlides/OpenSlides/pull/6052: please check if the two errors in the linked issue can no longer occur.

File: docs/datavalidator/check_json.py (hunk 1 above)
Comment (FinnStutzenstein) [reply docs/datavalidator/check_json.py:225]: The later checks rely on these checks to succeed. E.g. when checking relations, I can be sure, that all fields are there, I just have to check values.

**Comment by FinnStutzenstein** [comment]
No, these issues belong to the models validator which I didn't change

**Comment by jsangmeister** [comment]
Ahh ok, my bad

**Pull request merged**